x-pack/platform/plugins/shared/observability_ai_assistant/server/routes/functions/route.ts

```diff
@@ -81,23 +84,18 @@ const getFunctionsRoute = createObservabilityAIAssistantServerRoute({
 const functionRecallRoute = createObservabilityAIAssistantServerRoute({
   endpoint: 'POST /internal/observability_ai_assistant/functions/recall',
   params: t.type({
-    body: t.intersection([
-      t.type({
-        queries: t.array(
-          t.intersection([
-            t.type({
-              text: t.string,
-            }),
-            t.partial({
-              boost: t.number,
-            }),
-          ])
-        ),
-      }),
-      t.partial({
```

Comment:
@sorenlouv I don't think we are using this anywhere anymore, but can you confirm perhaps?